include/mbedtls/ssl.h

```diff
@@ -2269,7 +2269,7 @@ void mbedtls_ssl_conf_curves( mbedtls_ssl_config *conf,
 #if defined(MBEDTLS_KEY_EXCHANGE__WITH_CERT__ENABLED)
 /**
  * \brief          Set the allowed hashes for signatures during the handshake.
- *                 (Default: all available hashes except MD5.)
+ *                 (Default: all SHA2 hashes, largest first.)
```

Comment:
I don't think that correct, in 2.16 `MBEDTLS_TLS_DEFAULT_ALLOW_SHA1_IN_KEY_EXCHANGE` is enabled in the default `config.h`.

Reply:
Note: "all available hashes except MD5" was a bit misleading too, as we don't include RIPEMD-160...